src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -39,7 +39,11 @@ class DraftAdjudicationService(
     val draftAdjudication = DraftAdjudication(
       prisonerNumber = prisonerNumber,
       agencyId = agencyId,
-      incidentDetails = IncidentDetails(locationId = locationId, dateTimeOfIncident = dateTimeOfIncident)
+      incidentDetails = IncidentDetails(
+        locationId = locationId,
+        dateTimeOfIncident = dateTimeOfIncident,
+        handoverDeadline = dateCalculationService.calculate48WorkingHoursFrom(dateTimeOfIncident)
```

Comment:
We are calculating the deadline when it is saved so we don't have to call the API on every request, even when the API request doesn't actually use it.